### PR TITLE
oc: better new-app suggestions

### PR DESCRIPTION
**Pull request by kargakis**
@smarterclayton @bparees PTAL

### CODE REVIEW

File: pkg/cmd/cli/cmd/newapp.go [hunk 1]
```diff
@@ -286,9 +286,9 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 				}
 			}
 			if triggered {
-				fmt.Fprintf(out, "%sBuild scheduled for %q, use 'oc logs' to track its progress.\n", indent, t.Name)
+				fmt.Fprintf(out, "%sBuild scheduled for %q, use 'oc logs -f bc/%s' to track its progress.\n", indent, t.Name, t.Name)
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/newapp.go]: Name is redundant, just say "build scheduled, use 'oc logs -f bc/%s'"

File: pkg/cmd/cli/cmd/newapp.go [hunk 2]
```diff
@@ -286,9 +286,9 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
...
 			} else {
-				fmt.Fprintf(out, "%sBuild config %q does not include any automatic triggers, use 'oc start-build' to start a build.\n", indent, t.Name)
+				fmt.Fprintf(out, "%sBuild config %q does not include any automatic triggers, use 'oc start-build %s' to start a build.\n", indent, t.Name, t.Name)
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/newapp.go]: Same, try never to say the same name in the message twice.

File: pkg/cmd/cli/cmd/newapp.go (hunk 1 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/newapp.go]: ok

File: pkg/cmd/cli/cmd/newapp.go (hunk 2 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/newapp.go]: ok

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by kargakis** [comment]
[merge]

**Comment by kargakis** [comment]
https://github.com/openshift/origin/issues/8039 flake

**Comment by kargakis** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Comment by kargakis** [comment]
@smarterclayton pretty sure @liggitt aborted this for the rebase. Shouldn't we wait?

**Comment by kargakis** [comment]
> @smarterclayton pretty sure @liggitt aborted this for the rebase. Shouldn't we wait?

oh, the rebase just landed, nvmd

**Pull request merged**